collect_app/src/main/java/org/odk/collect/android/database/DatabaseObjectMapper.kt

```diff
@@ -20,11 +22,17 @@ object DatabaseObjectMapper {
         values.put(DatabaseFormColumns.DESCRIPTION, form.description)
         values.put(DatabaseFormColumns.JR_FORM_ID, form.formId)
         values.put(DatabaseFormColumns.JR_VERSION, form.version)
-        values.put(DatabaseFormColumns.FORM_FILE_PATH, StoragePathProvider.getRelativeFilePath(formsPath, form.formFilePath))
+        values.put(
```

Comment:
Yeah I agree, and I'll try and separate formatting existing code from new things in my commits more.

One thing to note is I'm just using Android Studio's "Reformat Code", (CMD+OPTION+L on Mac) which I basically hit after I write any code from muscle memory now. The default settings (which come from the [Kotlin Style Guide](https://kotlinlang.org/docs/coding-conventions.html)) seem to be much more aggressive than the Java ones, so there is much likelihood of adding a small piece of code to a file, autoformatting and seeing a lot of change if the file wasn't formatted correctly before. It seems the main culprit is max line length which Android Studio automatically reformats but ktLint seem to care about. I'll have a quick look at adding a rule for ktLint to fail on max line length, so it's less likely that we commit or merge incorrectly formatted code.

Reply:
Ah ok. The problem is that we're not using the [Android Kotlin Style Guide](https://developer.android.com/kotlin/style-guide) with ktlint but we can easily configure it use that instead of the Jetbrains style guide. Then it should match what Android Studio auto formats to (by default). I'll make another PR for that as it'll introduce a bunch of formatting.